File: src/models/comment.go
package models

import (
	"time"
)

type Comment struct {
	Id               string
	CreationTime     time.Time
	ModificationTime time.Time
	Content          string
	Uri              string
	Author           string
	Email            string
	Link             string
	Extra            string
}

type CommentQuery struct {
	Id               string
	CreationTime     time.Time
	ModificationTime time.Time
	Content          string
	Uri              string
	Author           string
	Email            string
	Link             string
	Offset           int
	Limit            int
}

File: sdk/Loment/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Loment.Models
{
    public class Comment
    {
        public string Id { get; set; } = string.Empty;

        public DateTimeOffset CreationTime { get; set; }

        public DateTimeOffset ModificationTime { get; set; }

        public string Content { get; set; } = string.Empty;

        public string Uri { get; set; } = string.Empty;

        public string Author { get; set; } = string.Empty;

        public string Email { get; set; } = string.Empty;

        public string Link { get; set; } = string.Empty;

        public string Extra { get; set; } = string.Empty;
    }
}
